src/pages/[locale]/buildings/[buildingSlug]/index.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <section v-if="building">
+    <BackButton
+      :use-history="false"
+      :to="buildingRoute({ buildingSlug: building.slug })"
+    />
+    <div class="default-layout__info building-layout">
+      <BuildingHeader class="building-layout__header" :building="building" />
+      <SpaceList class="building-layout__spaces" :spaces="spaces" />
+    </div>
+  </section>
+</template>
+
+<script setup lang="ts">
+import { useI18n } from "vue-i18n";
+import metaHead from "~/lib/meta-head";
+import { useStore } from "~/stores/store";
+import { useMapStore } from "~/stores/map";
+definePageMeta({
+  alias: "/:locale/gebouwen/:buildingSlug",
+});
+
+const { t } = useI18n();
+const store = useStore();
+const mapStore = useMapStore();
+const route = useRoute();
+const { buildingRoute } = useLocaleRoute();
+
+const building = computed(() =>
+  store.getBuildingBySlug(route.params.buildingSlug as string)
+);
+
+const spaces = computed(() =>
+  store.filteredSpaces.filter((space) => space.building === building.value)
+); //TODO: rly?
```

Comment:
what does this mean?

Reply:
I'm actually scared to filter comparing by reference. Do you think it looks safer to compare scalar/value-like things like `space.buildingNumber == building.value.number`? I mean OK probably it's one and the same object, but I have to read the other code to make sure it's so

Reply:
OK, I've encountered that it became an actual issue in Vue 3 (probably because of the new proxy model of reactivity), so I rewrote it to comparing the numbers.